### ISSUE TITLE
Layout shift on people metaspace when swithing space focus

### ISSUE THREAD

**Issue by germain-gg** [element-hq/element-web#25478]
### Steps to reproduce

1. Change space focus

#### What did you expect?

No layout shift

#### What happened instead?

https://github.com/vector-im/element-web/assets/769871/533eed9f-4b9e-46cc-b478-d6ad84b88ed9

### Application version

Element Nightly version: 2023053001
Olm version: 3.2.14

### Will you send logs?

No

**Comment by t3chguy** [comment]
@gsouquet does the same thing happen for the Home space? Can you also provide your browser & app version?

**Comment by gsouquet** [comment]
Yes, happens for all meta spaces. I updated the app version in the original description

**Comment by t3chguy** [comment]
Feels like a regression, doesn't happen on 

> Element Nightly version: 2023051101
Olm version: 3.2.14

**Comment by weeman1337** [comment]
Cannot be reproduced with the same desktop version or in the browser on https://develop.element.io/ .

Is the OS dependent? @t3chguy can you also reproduce the issue?

**Comment by t3chguy** [comment]
Though my video does show a 1px move on the Element Space when selecting it 🙃

**Comment by luixxiul** [comment]
For me it is reproduced not on 100% zoom level but less than that like 90%.


https://github.com/vector-im/element-web/assets/3362943/43fa5713-d00d-453d-9bb6-f9d352b500c3

**Comment by gsouquet** [comment]
Oh yes, my zoom level is 110% or 120%!
